pihole

```diff
@@ -300,6 +300,29 @@ Branches:
   checkout "$@"
 }
 
+tricorderFunc() {
+  if [ ! -p "/dev/stdin" ]; then
+    echo "Please do not call Tricorder directly."
+    exit 1
+  fi
+
+  if command -v openssl &> /dev/null; then
```

Comment:
Yeah, or connectivity is down. The debug used to have that check in it, I can't find it now, but may have been in a version that never merged. And it left a message that the server could not be connected to. (In which case, it just left the log on the local machine...) But that second part could be handled in debug.sh